.github/workflows/apply-konflux-manifests.yaml

```diff
@@ -0,0 +1,32 @@
+name: Apply Konflux Manifests
+on:
+  schedule:
+    - cron: "0 6 * * *" # Daily at 06:00.
+  workflow_dispatch: # Manual workflow trigger
```

Comment:
I think even on push when `.konflux` changes might make sense in the future (for PRs like https://github.com/openshift-knative/hack/pull/219)